File: trunk/PolarMath/src/org/macondo/polar/evaluation/spectrum/IC.java
package org.macondo.polar.evaluation.spectrum;

import org.macondo.polar.data.Training;
import org.macondo.polar.evaluation.Evaluation;

public class IC implements Evaluation<Double> {
	
	public Double evaluate(Training training) {
		double hf = training.evaluate(new HF());
		double lf = training.evaluate(new LF());
		double vlf = training.evaluate(new LF());
		
		return (lf + vlf) / hf;
	}

}

File: trunk/PolarMathCS/PolarMath/Evaluation/Spectrum/IC.cs
using PolarMath.Data;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolarMath.Spectrum
{
    public class IC : Evaluation<double>
    {
        public double evaluate(Training training)
        {
            double hf = training.evaluate( new HF() );
            double lf = training.evaluate( new LF() );
            double vlf = training.evaluate( new VLF() );

            return (lf + vlf) / hf;
        }
    }
}
